app/views/search/_result.html.erb

```diff
@@ -1,40 +1,28 @@
-<li>
+<li class="result">
   <h2 class="record-title">
     <span class="sr">Title: </span><%= link_to(result['title'], record_path(result['timdexRecordId'])) %>
   </h2>
 
-  <% result['contentType']&.each do |type| %>
-    <p>
-      <%= type %>
-    <p>
-  <% end %>
+  <p class="pub-info">
+    <span><%= result['contentType']&.each { |type| type['value'] }.join(' ; ') %></span>
+    <span>
+      <% result['dates']&.each do |date| %>
+        <%= date['value'] if date['kind'] == 'Publication date' %>
```

Comment:
I'm curious if we'll want to do something with additional dates. For now I agree with this for summary view as a starting point, although once we start considering different types of records I suspect this is an area that will get interesting.

Reply:
I don't intend additional work on this ticket on this, I was just thinking out loud.